### ISSUE TITLE
Does pokemon bank have compatibility issues?

### ISSUE THREAD

**Issue by neosnake87** [yellows8/ctr-httpwn#13]
I just renewed my pokemon bank which I haven't had since I've been using ctr-httpwn. I keep getting an error code when it is connecting to the server and what I've come up with from the nintendo forums is that I should get rid of my custom dns and set dns to auto so I did that but I am still getting the error message. I am wondering if ctr-httpwn is the issue. and if so would there be a work around?

**Comment by yellows8** [comment]
What error-code?

**Comment by neosnake87** [comment]
Error code 006-0501 after selecting a game in pokemon bank before connecting to the server.

**Comment by yellows8** [comment]
#12 Hmmm.....

**Comment by yellows8** [comment]
Does eShop with a NNID linked work fine for you?

**Comment by neosnake87** [comment]
Yes, eShop works fine after activating ctr-httpwn. I noticed something earlier. The error message from what I've read online is related to the dns setting in the 3ds's internet options but when I changed it back to auto obtain and when I tried to change it to google dns, Ctr-httpwn gets deactivated so I have to reactivate it through hbl everytime I change my dns.

**Comment by yellows8** [comment]
Returning from system-settings == hardware-reboot, see README.

**Comment by neosnake87** [comment]
Could you be more specific? I read over the readme again but not sure what part of it you were referring to.

**Comment by yellows8** [comment]
"Once run successfully, ctr-httpwn will persist under the sysmodule until the sysmodule is terminated(shutdown/reboot/FIRM-launch)."

**Comment by neosnake87** [comment]
Okay, I understand now. Going to System Settings> Internet Options and returning to Home Menu actually reboots the 3ds. Didn't know that. I am still getting the error code when using pokebank. Any suggestions? Do you think that ctr-httpwn could be the problem? Also does ctr-httpwn modify the dns the 3ds uses in any way?

**Comment by yellows8** [comment]
ctr-httpwn doesn't touch DNS.

**Comment by neosnake87** [comment]
Okay, thank you for your time I'll have to try troubleshooting it with nintendo tech support.

**Comment by CodeJacker69** [comment]
Related to #12's conclusion that it was related to 10.3, I am running 10.3 on a 2DS and am getting the same issue. Is it possible to upgrade to certain firmware like 10.4 manually? I'm really curious if it's just a 10.3 issue or if it's just all firmware past a certain point.

**Comment by neosnake87** [comment]
Hey, Jahebi. This issue was never fully resolved for me. I confirmed that it was an issue with ctr-httpwn but I was unable to find an easy fix.. There is a fix that I used that worked wonders but wasn't quite an easy fix.. So here are your options. Easy way is to abandon your hax and homebrew launcher and upgrade to newest official firmware(which I did not do, for obvious reasons.) Or use this guide: https://github.com/Plailect/Guide/wiki to get custom firmware but there is a risk that you will brick but the risk is very low if you follow the instructions exactly. Takes a few hours minimum. Recommend you ask someone for help if it's too confusing or watch a similar youtube video guide. Custom Firmware is what worked for me.

**Comment by DChronosX** [comment]
I am now having this f-ing problem, 10.3 firmware, trying to access pokemon bank and getting the same f-ing error code when it's about to enter the server!

But here's the kicker: I originally set up menuhax and httpwn in order to enter the pokemon bank to get the regi-trio event pokemon and I had NO PROBLEM entering the bank then.  I didn't do anything different this time, I was on 10.3 firmware then, too, and tried my same DNS settings as before, as well as other settings... So why is it that I was able to enter the bank perfectly fine before back on march 30th? I entered into the box, checked my pokemon, transferred the regis into my game from the bank, it worked perfectly fine twice that day!

I'd update, but I want to stay on the firmware version that can still use that firmware downgrade thing at least until I can make a hardmod backup again.
